composer.json

```diff
@@ -42,9 +42,10 @@
     ],
     "license": "GPL-3.0-or-later",
     "require": {
-        "typo3/cms-core": "^10.4|^11.5",
-        "kitodo/presentation": "^5.0|dev-master",
-        "slub/slub-digitalcollections": "^4.0|dev-master"
+        "php": "7.4 - 8.3",
+        "typo3/cms-core": "^11.5.40|^12.4.21",
+        "kitodo/presentation": "1",
```

Comment:
Yes, PHPStan reports that as well. This is because SLUB Digital Collections needs to be updated: https://github.com/slub/slub_digitalcollections/pull/94.

Reply:
You're right, that does seem to be an unintended change. I'll fix it right away!